components/Testimonial.js

```diff
@@ -1,35 +1,80 @@
-import Paragraph from "./typography/Paragraph";
+import ArrowLeft from '../components/icons/ArrowLeft'
+import ArrowRight from '../components/icons/ArrowRight'
+import { useState, useEffect } from 'react';
+import testimonials from './TestimonialData';
+
+const TestimonialCarousel = () => {
+  const [activeIndex, setActiveIndex] = useState(0);
+  const len = testimonials.length;
+
+  const goToPrevious = () => {
+    setActiveIndex((prevIndex) => (prevIndex === 0 ? len - 1 : prevIndex - 1));
+  };
+
+  const goToNext = () => {
+    setActiveIndex((prevIndex) => (prevIndex === len - 1 ? 0 : prevIndex + 1));
+  };
+
+  const goToIndex = (index) => {
+    setActiveIndex(index);
+  };
+
+  useEffect(() => {
+    const interval = setInterval(goToNext, 3000); 
+    return () => {
+      clearInterval(interval); 
+    };
+  }, []);
 
-export default function Testimonial({
-  className = '',
-  text,
-  authorName,
-  authorDescription,
-  authorAvatar,
-}) {
   return (
-    <li className={`py-4 px-4 sm:px-6 md:flex md:flex-row md:py-4 md:pr-0 ${className}`}>
-      <blockquote className="mt-8 md:flex-grow md:flex md:flex-col">
-        <div className="relative text-lg leading-7 font-medium text-gray-600 md:flex-1">
-          <svg className="absolute top-0 left-0 transform -translate-y-2 h-8 w-8 text-primary-500" fill="currentColor" viewBox="0 0 32 32">
-            <path d="M9.352 4C4.456 7.456 1 13.12 1 19.36c0 5.088 3.072 8.064 6.624 8.064 3.36 0 5.856-2.688 5.856-5.856 0-3.168-2.208-5.472-5.088-5.472-.576 0-1.344.096-1.536.192.48-3.264 3.552-7.104 6.624-9.024L9.352 4zm16.512 0c-4.8 3.456-8.256 9.12-8.256 15.36 0 5.088 3.072 8.064 6.624 8.064 3.264 0 5.856-2.688 5.856-5.856 0-3.168-2.304-5.472-5.184-5.472-.576 0-1.248.096-1.44.192.48-3.264 3.456-7.104 6.528-9.024L25.864 4z" />
-          </svg>
-          <Paragraph className="relative pl-10 text-left">
-            {text}
-          </Paragraph>
-        </div>
-        <footer className="mt-6">
-          <div className="flex">
-            <figure className="flex-shrink-0 inline-flex rounded-full border-2 border-white">
-              <img className="h-12 w-12 rounded-full" src={authorAvatar} alt={authorName}/>
-            </figure>
-            <div className="ml-4 text-left">
-              <p className="text-base leading-6 font-bold text-gray-900">{authorName}</p>
-              <p className="text-base leading-6 font-medium text-primary-500">{authorDescription}</p>
+    <div className="relative flex flex-row justify-center items-center md:gap-4 overflow-x-hidden">
+      <div className="h-8 w-8 rounded-full bg-[#ddd] cursor-pointer mb-2 absolute left-0 z-10 top-1/2 transform -translate-y-1/2 opacity-50 md:opacity-100 flex justify-center items-center" onClick={goToPrevious}>
+       <ArrowLeft className='w-4'/>
+      </div>
+      <div className="flex flex-col justify-center items-center">
+        <div className="relative h-[300px] w-[300px] md:w-[600px] md:h-[450px] overflow-hidden">
```

Comment:
Don't specify height for the box, as you may not know how long can be testimonial text. Moreover, using `overflow-hidden` is not a good practice. You can allow `Read More` option inside box to expand the testimonial box.

Reply:
For some reason I cant set the height to auto... When I am doing height-auto or max-content the cards just disappear

Reply:
Weldone @akshatnema 

Hello @Lucif3r-in, where can I review the updated version of the updated version of the testimonial section? Or is it still the same link in the deploy preview column?

Reply:
Same link @Mayaleeeee

Reply:
1. The icon here is not centered in the middle.
![Screenshot 2023-10-02 134612](https://github.com/asyncapi/website/assets/105395613/8c0e31f4-20cc-4ccf-9ff9-8caa609f2a59)

2. You need to change the card background color to white and add elevation to match other cards on the website.
![Screenshot 2023-10-02 173125](https://github.com/asyncapi/website/assets/105395613/d5ba77e0-9a51-45b0-9b4c-2e9a07e5fa45)

Also, looking at the texts on the caard, I noticed it may be too small. However, I will recheck and update this comment before the end of the day.

Thank you @Lucif3r-in